rust/python_bindings/src/bindings.rs

```diff
@@ -570,9 +566,10 @@ impl Bindings {
         )?;
 
         let mut frontend_clone = self.frontend.clone();
-        let result = self
-            .runtime
-            .block_on(async { frontend_clone.get(request).await })?;
+        let result = py.allow_threads(move || {
```

Comment:
is it possible to free the GIL on inserts too?

Reply:
It doesn't materially impact anything since inserts are internally multithreaded. I benched it